modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/PostmanCollectionCodegen.java

```diff
@@ -0,0 +1,839 @@
+                        String exampleRef = entry.getValue().get$ref();
+                        Example example = this.openAPI.getComponents().getExamples().get(extractExampleByName(exampleRef));
+                        String exampleAsString = getJsonFromExample(example);
+
+                        items.add(new PostmanRequestItem(example.getSummary(), exampleAsString));
+                    }
+                } else if (codegenOperation.bodyParam.getSchema() != null) {
+                    // find in schema example
+                    String exampleAsString = formatJson(codegenOperation.bodyParam.getSchema().getExample());
+                    items.add(new PostmanRequestItem(codegenOperation.summary, exampleAsString));
+                } else {
+                    // example not found
+                    // get from schema
+                    items.add(new PostmanRequestItem(codegenOperation.summary, getJsonFromSchema(codegenOperation.bodyParam)));
+
+                }
+            }
+        } else {
+            // operation without bodyParam
+            items.add(new PostmanRequestItem(codegenOperation.summary, ""));
+        }
+
+        return items;
+    }
+
+    // from placeholders (ie {{VAR_1}}) create Postman variables
+    public List<PostmanRequestItem> createPostmanVariables(List<PostmanRequestItem> postmanRequests) {
+
+        for(PostmanRequestItem requestItem : postmanRequests) {
+            for(String var: extractPlaceholders(requestItem.getBody())) {
+                variables.add(new PostmanVariable()
+                        .addName(var)
+                        .addType("string")
+                        .addExample(""));
+            }
+        }
+
+        return postmanRequests;
+    }
+
+    // replace postmanGuid placeholder (ie {{UNIQUE_REFERENCE}}) with Postman formula {{$guid}}
+    List<PostmanRequestItem> setPostmanGuid(List<PostmanRequestItem> postmanRequests) {
+
+        for(PostmanRequestItem requestItem : postmanRequests) {
+            requestItem.setBody(requestItem.getBody().replace("{{" + postmanGuidPlaceholderName + "}}", "{{$guid}}"));
+        }
+
+        return postmanRequests;
+    }
+
+    // replace postman placeholder (ie {{ISO_TIMESTAMP}}) with Postman formula {{$isoTimestamp}}
+    List<PostmanRequestItem> setPostmanIsoTimestamp(List<PostmanRequestItem> postmanRequests) {
+
+        for(PostmanRequestItem requestItem : postmanRequests) {
+            requestItem.setBody(requestItem.getBody().replace("{{" + postmanIsoTimestampPlaceholderName + "}}", "{{$isoTimestamp}}"));
+        }
+
+        return postmanRequests;
+    }
+
+    /**
+     * Returns human-friendly help for the generator.  Provide the consumer with help
+     * tips, parameters here
+     *
+     * @return A string value for the help message
+     */
+    public String getHelp() {
+        return "Generates a postman JSON file";
```

Comment:
Shall we mention the supported Postman collection format (v2.1.0) here?

Reply:
Done